src/forms/InputTime.jsx

```diff
@@ -0,0 +1,120 @@
+// @flow
```

Comment:
We should add stories for this too instead of just having stories for `FieldsetTime`

Reply:
addressed by removing `FieldsetTime` story and adding lots of explicit variants for `InputTime` variants